### PR TITLE
support 2016.3

### PR DESCRIPTION
**Pull request by jbeckers**

### CODE REVIEW

**Comment by athiele** [comment]
Great! I will create a release for this soon.

File: build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ apply plugin: 'org.jetbrains.intellij'
 apply plugin: 'java'
 
 intellij {
-    version = '2016.2'
+    version = '2016.3'
```
Comment (jbeckers) [inline build.gradle]: It seems this won't work as long as 2016.3 is not released.

163.* should work

**Comment by athiele** [comment]
Hi, I already pushed something similar to master: 163.3512.7. This should be compatible with newer releases as well

**Comment by athiele** [comment]
So, the best solution was to fix the since-build value and tell gradle not to make a version of this plugin only be compatible for one major version of IntelliJ. See https://github.com/athiele/key-promoter-fork/commit/a7e126a807e8fca88f952092429f7d37a0c635e4

**Pull request merged**